java/ql/test/library-tests/frameworks/apache-commons-lang3/flow.ql

```diff
@@ -17,11 +29,18 @@ class Conf extends TaintTracking::Configuration {
 class HasFlowTest extends InlineExpectationsTest {
   HasFlowTest() { this = "HasFlowTest" }
 
-  override string getARelevantTag() { result = "hasTaintFlow" }
+  override string getARelevantTag() { result = ["hasTaintFlow", "hasValueFlow"] }
 
   override predicate hasActualResult(Location location, string element, string tag, string value) {
     tag = "hasTaintFlow" and
-    exists(DataFlow::Node src, DataFlow::Node sink, Conf conf | conf.hasFlow(src, sink) |
+    exists(DataFlow::Node src, DataFlow::Node sink, TaintFlowConf conf | conf.hasFlow(src, sink) |
```

Comment:
Might as well add `and not any(ValueFlowConf conf | conf.hasFlow(src, sink))` to make the expected output shorter - there's no need to report taint-flow if we have value-flow as that will always be the case.

Reply:
done